presto-native-execution/pom.xml

```diff
@@ -232,6 +233,12 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>com.facebook.presto</groupId>
+            <artifactId>presto-spark-classloader-spark${dep.pos.classloader.module-name.suffix}</artifactId>
```

Comment:
@wangkewen The `presto-native-exection/pom.xml` and `presto-spark-base/pom.xml` need not have the `***-spark2` and `***-spark3` dependencies. They can simply depend on the `presto-spark-classloader-interface` which internally pulls in correct `2` v/s `3` dependency.

Or am I missing something ?